test/one/profiler/test/TestProcess.java

```diff
@@ -96,7 +103,10 @@ private List<String> buildCommandLine(Test test, Os currentOs) {
             cmd.add(substituteFiles(String.join(";", sh)));
         } else {
             cmd.add(System.getProperty("java.home") + "/bin/java");
-            cmd.add("-cp");
+            if (needsDynamicAgentLoadingFlag()) {
+                cmd.add("-XX:+EnableDynamicAgentLoading");
+            }
```

Comment:
Why is this option needed? `EnableDynamicAgentLoading` is true even in JDK 24.
For the cases when dynamic agent loading is disabled, I'll implement a different solution in jattach.

Reply:
Ok, I'll remove it